plugins/arcgis/service/src/ObservationProcessor.ts

```diff
@@ -151,6 +164,19 @@ export class ObservationProcessor {
         return config
     }
 
+    public sanitizeFeatureService(config: FeatureServiceConfig, type: AuthType): FeatureServiceConfig {
```

Comment:
Sanitization should happen in the context of web responses.  This can live with the web route definitions.